app/src/main/cpp/skyline/services/am/controller/ILibraryAppletCreator.h

```diff
@@ -20,14 +20,21 @@ namespace skyline::service::am {
         Result CreateLibraryApplet(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response);
 
         /**
-         * @brief Creates an IStorage that can be used by the application
+         * @brief Creates an IStorage that can be used by the application, backed by service-allocated memory
          * @url https://switchbrew.org/wiki/Applet_Manager_services#CreateStorage
          */
         Result CreateStorage(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response);
 
+        /**
+         * @brief Creates an IStorage that can be used by the application, that accesses the supplied transfer memory
```

Comment:
`backed by` would be better terminology and more uniform as the application is renting memory to the service to create an `IStorage` out of.